.github/workflows/push-foundation-repo.yml

```diff
@@ -0,0 +1,16 @@
+name: Sync Cardinal-Cryptography repo with Aleph-Zero-Foundation repo
+
+on:
+  push:
+    branches: [master]
+
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Push to Aleph-Zero-Foundation
+        run: |
+          git remote add alephzero https://x-access-token:${{ secrets.SYNCAZF }}@github.com/aleph-zero-foundation/AlephBFT
+          git push alephzero
```

Comment:
I'm pretty sure you can just do this in one step, just writing the address above instead of `alephzero` here.

Reply:
Done